py/desispec/quicklook/util.py

```diff
@@ -5,51 +5,60 @@
 put pipeline related utility functions here.
 """
 
+import os
 import numpy as np
 import yaml
+from desispec.quicklook import qllogger
 
-def merge_QAs(qaresult):
+def merge_QAs(qaresult,config):
```

Comment:
Are we going to merge the same way between QL and offline?

If so, this method should be in a different place.

Reply:
We also concluded merge_QAs was not the place to do this, but perhaps for different reasons.  

In QL the merge is very brief, being specified just by the schema lines above, and gathering the individual QA dictionaries as the pipeline is executed.  So the util.py code is no longer executed, and it would be removed either when we merge the PR or in a later branch.

Perhaps the most important thing for the merged QA implementation is to have a common place for the definition of the schema, to avoid diverging implementations.  Let me know if you agree.

Reply:
This method is obsolete and should be removed.